playwright-tests/tests/mentorship-dropdown.page.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { MentorshipDropdownPage } from '@pages/mentorship-dropdown.page';
```

Comment:
Please, include this test case as part of `navigation.page.spec.ts`. It will maintain cleaner code and will be easier to navigate.